packages/react/src/oxygen-ui-react-auth-components/SignInButton/sign-in-button.scss

```diff
@@ -0,0 +1,21 @@
+.oxygen-sign-in-button {
+  margin: 16px 0;
+}
+
+.oxygen-sign-in-button-social {
+  background: white !important;
+  color: black !important;
```

Comment:
Is marking these css properties as !important intentional? wouldn't end users be able to override the component styles?

Reply:
It needs to be updated.
Addressed this with 39e975757996aea484c759adb1d33e4347b5588a.